src/libs/elektra/opmphm_vheap.c

```diff
@@ -6,7 +6,7 @@
  * To construct a max heap the comparison function VheapComp (a, b), must return
  * 1 on a > b and 0 otherwise. For a min heap 1 on a < b and 0 otherwise.
  *
- * @ingroup vheap
```

Comment:
Maybe we can add a group "Internal Datastructures" and vheap, vstack and trie are subgroups?